tensor-groups/yolo-v8-pose-out.md

```diff
@@ -0,0 +1,59 @@
+# Classification
+- tensor-group: yes
+- layer-type: output
+- use-case: pose-estimation
+
+# Description
+
+Variant of YOLOv8-Pose outputs. This tensor group contains three tensors that 
+together describe the predictions of yolov8-pose. The “8400” comes from the total 
+number of grid-cell predictions at strides [8,16,32] for a 640 × 640 input:
```

Comment:
Is it always 640x640? I'm guessing the "8400" is a variable number here, no ?

Reply:
okay, I exported the model with 640x640 so the input and output shape are fixed for this model. Although we can make it dynamic as well which allows feeding input with different shapes and that should change the output shape as well.